app/src/main/java/com/devshish/internship/presentation/ui/lyrics/LyricsFragment.kt

```diff
@@ -45,6 +46,11 @@ class LyricsFragment : Fragment(R.layout.fragment_lyrics) {
         }
 
         with(viewModel) {
+            noInternetConnection.observe(viewLifecycleOwner) {
+                Snackbar.make(requireView(), R.string.internet_connection_absent, Snackbar.LENGTH_INDEFINITE)
```

Comment:
same here

Reply:
Also R.string.internet_connection_absent change it to "Something gone wrong" or something like that